prosody/Dockerfile

```diff
@@ -1,11 +1,24 @@
 FROM jitsi/base
 
+ADD https://raw.githubusercontent.com/jitsi/jitsi-meet/05b7e6facc25422fba6a4c264e8f42411910019d/resources/prosody-plugins/token/util.lib.lua /prosody-plugins/token/util.lib.lua
+ADD https://raw.githubusercontent.com/jitsi/jitsi-meet/05b7e6facc25422fba6a4c264e8f42411910019d/resources/prosody-plugins/mod_token_verification.lua /prosody-plugins/mod_token_verification.lua
+ADD https://raw.githubusercontent.com/jitsi/jitsi-meet/05b7e6facc25422fba6a4c264e8f42411910019d/resources/prosody-plugins/mod_auth_token.lua /prosody-plugins/mod_auth_token.lua
+
 RUN \
 	apt-dpkg-wrap apt-get update && \
+	apt-dpkg-wrap apt-get install -y lua5.2 liblua5.2-dev libssl1.0-dev luarocks gcc git && \
 	apt-dpkg-wrap apt-get install -t stretch-backports -y prosody && \
-	apt-cleanup && \
 	rm -rf /etc/prosody
 
+RUN \
+  luarocks install lua-cjson 2.1.0-1 && \
```

Comment:
I hate this cjson thing so much I took a closer look. It's totally unmaintained and there is no fork which supports Lua 5.2 properly (unless you se an old version which didn't implement certain feature). So it doesn't seem very future proof.

Since the luajwtjitsi module is ours, I think we can just swap the cjson module for another better JSON library.

In addition, luacrypto is super-deprecated and it's recommended to use luaossl now.

I'm going to try to update luajwtjitsi so it uses those 2 so this can be greatly simplified. You don't have to do anything, but thanks for listening to my rant :-)